hack/start-minikube.sh

```diff
@@ -2,13 +2,28 @@
 set -e
 set -x
 
+# Figure out where we are being run from.
+# This relies on script being run from:
+#  - ${PROJECT_ROOT}/hack/install-tackle.sh
+#  - ${PROJECT_ROOT}/bin/install-tackle.sh
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__root="$(cd "$(dirname "${__dir}")" && pwd)"
+__repo="$(basename "${__root}")"
+__bin_dir="${__root}/bin"
+__os="$(uname -s | tr '[:upper:]' '[:lower:]')"
+__arch="$(uname -m | sed 's/x86_64/amd64/' | sed 's/aarch64/arm64/')"
+
+# Update PATH for execution of this script
+export PATH="${__bin_dir}:${PATH}"
```

Comment:
If L14 don't need exporting, then why is L17 export needed?

Reply:
The export is not needed here (cf. https://github.com/konveyor/operator/pull/275#discussion_r1380130973). The general difference here is that `${PATH}` is an environment variable (by the export) and `${__arch}` (and the others) are shell variables.

Reply:
Then keep it unexported so it don't alter user environment?

Reply:
No, the concept is very different: We are on Unix, not on Windows. Environment-Variables are only inherited by new sub-processes of the running process. They never ever affect the environment of any other process (even of the same user). Shell-variables are only relevant to the current shell/script. They are never inherited by new sub-processes (even if the sub-process itself is a shell).